packages/manager/apps/telecom/src/app/telecom/pack/xdsl/pack-xdsl.routing.js

```diff
@@ -28,28 +28,7 @@ export default /* @ngInject */ ($stateProvider) => {
     url: '/xdsl/:serviceName/lines/:number',
     views: {
       'packView@telecom.packs': 'packXdsl',
-      'xdslView@telecom.packs.pack.xdsl': {
-        controller: 'XdslAccessCtrl',
-        controllerAs: 'XdslAccess',
-        templateUrl: 'app/telecom/pack/xdsl/access/pack-xdsl-access.html',
-      },
-    },
```

Comment:
Deleting the view ?

Reply:
It's not deleting, just used the component

Reply:
My bad, I didn't saw the last line in green below :)